testdata/oracle/complex/bulk-operations.json

```diff
@@ -0,0 +1,16 @@
+{
+    "input": "DECLARE TYPE EmpTabTyp IS TABLE OF employees%ROWTYPE INDEX BY PLS_INTEGER; emp_tab EmpTabTyp; BEGIN SELECT * BULK COLLECT INTO emp_tab FROM employees; FORALL i IN emp_tab.FIRST .. emp_tab.LAST SAVE EXCEPTIONS UPDATE employees SET test = test * 1.05 WHERE employee_id = emp_tab(i).employee_id; END;",
+    "outputs": [
+      {
+        "expected": "DECLARE TYPE EmpTabTyp IS TABLE OF employees % ROWTYPE INDEX BY PLS_INTEGER; emp_tab EmpTabTyp; BEGIN SELECT * BULK COLLECT INTO emp_tab FROM employees; FORALL i IN emp_tab.FIRST . . emp_tab.LAST SAVE EXCEPTIONS UPDATE employees SET test = test * ? WHERE employee_id = emp_tab ( i ) . employee_id; END",
```

Comment:
It would be correct to keep `;` in the end. The program wouldn't execute otherwise.

Reply:
updated. the trailing semicolon is controlled by `KeepTrailingSemicolon` and disabled by default.